### DISCUSSION TITLE
How do i get table of contents in one separate page and the heading file in one single page when rendering into pdf ??

### DISCUSSION THREAD

**Discussion by Siddhu2502** [quarto-dev/quarto-cli#8754] [Q&A]
### Description

---
title: "Assignment 1"
subtitle: "RL Assignment 1"
authors: 
 - "Siddharth D"
 - "CB.EN.U4AIE21064"
abstract: " In this paper we will compare 2 RL algorithms A2C and DQN and alalyze how they perform on a simple environment which a cart pole environment provided by OpenAI gym. We will neatly understand all the contents of the workings of each of the algorithms and then compare their performance."
format:
  pdf:
    toc: true
    toc-depth: 3
    number-sections: true
    colorlinks: true
---

I am trying to render my project as pdf but the problem is that i want the author and abstract in one page and then the TOC in another page basically i want a custom front page but as of now i am just cutting the pdf upto TOC and then reattaching with my custom page form pdf editor but is there a way around it ik there is a way around but i am not able to figure it out ? what should i do to acheive that 

1) custom front page only just one page 
2) getting TOC in new page (only the TOC)

**Comment by Siddhu2502** [comment]
@kozo2 @cwickham @aborruso @msarahan @mcanouil  anyone to help ?

**Reply by mcanouil** [reply]
Please don't ping people like that, especially after a weekend.

The best way to get a quick answer is to give a reproducible example as plain text or as a Git repository.

This being said, did you read the documentation about template and partials?

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
